packages/admin/admin/src/common/FieldSet.tsx

```diff
@@ -0,0 +1,147 @@
+
+const styles = (theme: Theme) =>
+    createStyles<FieldSetClassKey, FieldSetProps>({
+        header: {
+            display: "flex",
+            flexDirection: "row-reverse",
+            padding: "0 10px",
+            height: "80px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "0 20px",
+            },
+        },
+        headerColumn: {
+            display: "flex",
+            flexDirection: "column",
+            alignItems: "flex-start",
+            alignContent: "center",
+            padding: "0px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "10px",
+            },
+        },
+        title: {
+            display: "flex",
+            alignItems: "center",
+            fontWeight: theme.typography.fontWeightMedium,
+            fontSize: "14pt",
+            textTransform: "uppercase",
+            color: theme.palette.text.primary,
+        },
+        supportText: {
+            fontSize: "10pt",
+            lineHeight: "18px",
+            color: theme.palette.text.secondary,
+        },
+        endAdornment: { display: "flex", alignItems: "center" },
+        placeholder: {
+            flexGrow: 1,
+            boxSizing: "inherit",
+            userSelect: "none",
+        },
+        children: {
+            display: "flex",
+            flexDirection: "column",
+            borderTop: `solid ${theme.palette.divider}`,
+            padding: "20px",
+            [theme.breakpoints.up("sm")]: {
+                padding: "40px",
+            },
+            "&$disablePadding": {
+                padding: "0px",
+            },
+        },
+        disablePadding: {},
+    });
+
+function FieldSet({
+    title,
+    supportText,
+    endAdornment,
+    children,
+    collapsible = true,
+    initiallyExpanded = true,
+    disablePadding = false,
+    componentsProps,
+    classes,
+}: React.PropsWithChildren<FieldSetProps> & WithStyles<typeof styles>): React.ReactElement {
+    const [expanded, setExpanded] = React.useState(initiallyExpanded);
+
+    const handleChange = (event: React.SyntheticEvent, isExpanded: boolean) => {
+        setExpanded(isExpanded);
+    };
+
+    return (
+        <MuiAccordion
+            expanded={expanded}
+            onChange={
+                collapsible
+                    ? handleChange
+                    : () => {
+                          /* do nothing */
+                      }
+            }
+        >
+            <MuiAccordionSummary
+                classes={{ root: classes.header }}
+                expandIcon={collapsible && <ArrowForwardIosSharpIcon />}
+                {...componentsProps?.fieldSetSummary}
+            >
+                <div className={clsx(classes.headerColumn)}>
+                    <div className={clsx(classes.title, expanded && classes.disablePadding)}>{title}</div>
```

Comment:
Why is the `disablePadding` class dependent on `expanded`?  
Should there maybe be a separate `expanded` class? Or should this be dependent on `disablePadding`?